samples/nrf_desktop/configuration/nrf52_pca20037/app_ZReleaseMCUBoot.conf

```diff
@@ -0,0 +1,125 @@
+################################################################################
+# Application Configuration
+
+CONFIG_DESKTOP_REPORT_DESC="configuration/common/hid_report_desc.c"
+
+CONFIG_DESKTOP_HID_KEYBOARD=y
+CONFIG_DESKTOP_HID_STATE_ENABLE=y
+
+CONFIG_DESKTOP_BUTTONS_ENABLE=y
+
+#CONFIG_DESKTOP_LED_ENABLE=y
+#CONFIG_DESKTOP_LED_COUNT=1
+#CONFIG_DESKTOP_LED_COLOR_COUNT=3
+
+CONFIG_DESKTOP_POWER_MANAGER_ENABLE=y
+
+CONFIG_DESKTOP_BLE_ADVERTISING_ENABLE=y
+CONFIG_DESKTOP_BLE_DIRECT_ADV=n
+CONFIG_DESKTOP_BLE_FAST_ADV=y
+CONFIG_DESKTOP_BLE_SWIFT_PAIR=y
+CONFIG_DESKTOP_BLE_SHORT_NAME="Kbd nRF52"
+
+CONFIG_DESKTOP_CONFIG_ENABLE=y
+CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
+CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_ENABLE=y
+
+################################################################################
+# Zephyr Configuration
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1280
+CONFIG_ISR_STACK_SIZE=1536
+CONFIG_MAIN_STACK_SIZE=1024
+CONFIG_IDLE_STACK_SIZE=128
+CONFIG_BT_RX_STACK_SIZE=2048
+
+CONFIG_BOOT_BANNER=n
+
+CONFIG_NUM_COOP_PRIORITIES=10
+CONFIG_NUM_PREEMPT_PRIORITIES=11
+
+CONFIG_HEAP_MEM_POOL_SIZE=512
+CONFIG_HEAP_MEM_POOL_MIN_SIZE=32
+
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+
+CONFIG_SYS_PM_POLICY_APP=y
+
+CONFIG_GPIO=y
+
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+CONFIG_FCB=y
+
+CONFIG_SETTINGS=y
+CONFIG_SETTINGS_FCB=y
+
+CONFIG_REBOOT=y
+
+CONFIG_SPEED_OPTIMIZATIONS=y
+
+CONFIG_PWM=y
+CONFIG_PWM_NRF5_SW=n
+CONFIG_PWM_NRFX=y
+
+CONFIG_PWM_0=y
+
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC=y
+
+CONFIG_BT=y
+CONFIG_BT_DEBUG_NONE=y
+CONFIG_BT_SETTINGS=y
+CONFIG_BT_SETTINGS_CCC_STORE_ON_WRITE=y
+CONFIG_BT_SMP=y
+CONFIG_BT_SIGNING=y
+CONFIG_BT_TINYCRYPT_ECC=y
+CONFIG_BT_MAX_PAIRED=3
+CONFIG_BT_ID_MAX=3
+CONFIG_BT_AUTO_PHY_UPDATE=n
+
+CONFIG_BT_DEVICE_NAME="Keyboard nRF52 Desktop"
+CONFIG_BT_DEVICE_APPEARANCE=962
```

Comment:
Keyboard have appearance 961 (0x3C1) 
https://www.bluetooth.org/DocMan/handlers/DownloadDoc.ashx?doc_id=429632

Reply:
missed that, thanks

Reply:
Fixed